### ISSUE TITLE
[Feature Request] Ability to detect "legit" fast-break

### ISSUE THREAD

**Issue by TheosisOfficial** [konsolas/AAC-Issues#2166]
### Issue description
"Legit" fast-break is a hack where players skip the cooldown there is between mining blocks (there is no cooldown when the block breakage is instant, but there is otherwise). This isn't a huge deal normally, but the time it saves over a long period of time on prison servers for example could be a lot, so it should be detected if possible.
### Core information
**Server version**: 1.16.5

**AAC version**: 5.0.10

**ProtocolLib version**: 4.6.0-SNAPSHOT-b493
### Configuration
No modifications that would affect the current behaviour of what I'm describing.

**Comment by Janmm14** [comment]
I think it was part of AAC 1/2 and got remvoed during AAC3 rewrite.

**Comment by TheosisOfficial** [comment]
> I think it was part of AAC 1/2 and got removed during AAC3 rewrite.

Yeah, I remember you telling me this in the Discord. It would be cool to see it re-added!
